mw-webapp/src/component/modal/PromptModalContent.tsx

```diff
@@ -67,32 +74,43 @@ export const PromptModalContent = (props: PromptModalContentProps) => {
 
   return (
     <div onKeyDown={handleEnter}>
-      <Input
-        type="text"
-        placeholder={props.placeholder}
-        value={inputValue}
-        autoFocus={true}
-        onChange={setInputValue}
-        typeInput={InputType.Line}
-      />
-      <HorizontalContainer className={styles.buttons}>
-        <DialogClose asChild>
-          <Button
-            value={props.cancelButtonValue}
-            onClick={props.close}
-          />
-        </DialogClose>
-
-        <DialogClose asChild>
-          <Button
-            ref={onOkRef}
-            value={props.okButtonValue}
-            onClick={() => props.onOk(inputValue)}
-            buttonType={ButtonType.PRIMARY}
-            isDisabled={inputValue.trim() === ""}
-          />
-        </DialogClose>
-      </HorizontalContainer>
+      <VerticalContainer className={styles.container}>
+
+        {props.title &&
+        <Title
+          level={HeadingLevel.h2}
+          text={props.title}
+          placeholder=""
+        />
+        }
+
```

Comment:
What do you think about using Dialog.title in Modal component instead of adding Title component for each content separately? Just for discuss)

Reply:
I don't think it's a good idea because the title is integral to the content and should logically be passed along with the rest of the content. If I understand correctly